tools/create_release.py

```diff
@@ -183,7 +183,7 @@ async def main():
     icon_path = randovania.get_icon_path()
     shutil.copyfile(icon_path, icon_path.with_name("executable_icon.ico"))
 
-    if (secret := os.environ.get("OBFUSCATOR_SECRET")) is not None:
+    if (secret := os.environ.get("OBFUSCATOR_SECRET")) is not None and len(secret) >= 1 and not secret.isspace():
```

Comment:
maybe this?
```python
secret = os.environ.get("OBFUSCATOR_SECRET", "")
if secret.trim() and not secret.isspace():
```

Reply:
What is `trim` supposed to be for a function? Do you mean strip?

Reply:
probably, too used to different names in different languages

Reply:
doesn't work then.
Stripping an empty string, results in an empty string. and isspace only works on strings with length bigger than 0.
```py
>>> s = ""
>>> s.strip().isspace()
False
>>> 
```

Reply:
I did suggest both conditions?

Reply:
oh, i got switched up by it. forget sometimes that a string itself can be truthy.